### PR TITLE
fix: Int values from the CLI

### LINKED ISSUE #2722: CLI: Unable to set values other than strings
### Describe the bug

At the moment it is **not posibble** to set anything other than a string via the CLI.

### Steps to reproduce

1. Create a distro and a profile
2. Create a system
3. Try to set e.g. the Baud rate or the # of virt CPUs of the system

````bash
❯ cobbler system add --name stest2 --profile ptest --serial-baud-rate=240 
Usage: cobbler [options]

cobbler: error: option --serial-baud-rate: invalid choice: '240' (choose from '', '2400', '4800', '9600', '19200', '38400', '57600', '115200')

❯ cobbler system add --name stest2 --profile ptest --serial-baud-rate=2400
exception on server: serial baud rate needs to be of type int or enums.BaudRates

❯ cobbler system add --name stest2 --profile ptest --virt-cpus=1            
exception on server: virt_cpus needs to be an integer

````

### Expected behavior
All values should be settable via the CLI.

### Cobbler version

Current master branch.

````bash
$ cobbler version
 source: ab950916, Mon Jul 26 17:27:06 2021 +0200
  build time: Wed Jul 28 09:19:45 2021
````

### Operating system
openSUSE Tumbleweed, SLES 15-SP3

### Cobbler log

````bash
(...)
[Thread-173] 2021-07-29T09:44:33 - INFO | Exception occurred: <class 'TypeError'>
[Thread-173] 2021-07-29T09:44:33 - INFO | Exception value: serial baud rate needs to be of type int or enums.BaudRates
[Thread-173] 2021-07-29T09:44:33 - INFO | Exception Info:
  File "/usr/local/lib/python3.8/site-packages/cobbler/remote.py", line 3514, in _dispatch
    return method_handle(*params)

  File "/usr/local/lib/python3.8/site-packages/cobbler/remote.py", line 1942, in xapi_object_edit
    self.modify_item(object_type, handle, key, value, token)

  File "/usr/local/lib/python3.8/site-packages/cobbler/remote.py", line 1696, in modify_item
    setattr(obj, attribute, arg)

  File "/usr/local/lib/python3.8/site-packages/cobbler/items/system.py", line 1760, in serial_baud_rate
    self._serial_baud_rate = validate.validate_serial_baud_rate(baud_rate)

  File "/usr/local/lib/python3.8/site-packages/cobbler/validate.py", line 530, in validate_serial_baud_rate
    raise TypeError("serial baud rate needs to be of type int or enums.BaudRates")
(...)
[Thread-193] 2021-07-29T09:49:02 - INFO | Exception occurred: <class 'TypeError'>
[Thread-193] 2021-07-29T09:49:02 - INFO | Exception value: virt_cpus needs to be an integer
[Thread-193] 2021-07-29T09:49:02 - INFO | Exception Info:
  File "/usr/local/lib/python3.8/site-packages/cobbler/remote.py", line 3514, in _dispatch
    return method_handle(*params)

  File "/usr/local/lib/python3.8/site-packages/cobbler/remote.py", line 1942, in xapi_object_edit
    self.modify_item(object_type, handle, key, value, token)

  File "/usr/local/lib/python3.8/site-packages/cobbler/remote.py", line 1696, in modify_item
    setattr(obj, attribute, arg)

  File "/usr/local/lib/python3.8/site-packages/cobbler/items/system.py", line 1380, in virt_cpus
    self._virt_cpus = validate.validate_virt_cpus(num)

  File "/usr/local/lib/python3.8/site-packages/cobbler/validate.py", line 497, in validate_virt_cpus
    raise TypeError("virt_cpus needs to be an integer")
````

### PR DESCRIPTION
**Pull request by nodeg**
This partially (for ints/floats) fixes #2722.

### CODE REVIEW

**Comment by nodeg** [comment]
This test definitely passed in the docker container. I will look into it.

**Comment by SchoolGuy** [comment]
@nodeg Remember to fix the:

```
cobbler/validate.py:332:5: E265 block comment should start with '# '
```

File: tests/items/profile_test.py [hunk 1]
```diff
@@ -301,9 +301,9 @@ def test_virt_disk_driver(value, expected_exception):
 
 
 @pytest.mark.parametrize("value,expected_exception", [
-    ("", pytest.raises(ValueError)),
+    ("",  pytest.raises(ValueError)),
```
Comment (SchoolGuy) [inline tests/items/profile_test.py:304]: There slipped an additional whitespace. Would be cool if you could remove that.

Comment (nodeg) [reply tests/items/profile_test.py]: Amende.

**Review by SchoolGuy — APPROVED** [review]
LGTM

**Comment by SchoolGuy** [comment]
I don't know why Codacy has the false positives but they are definitely incorrect.

**Pull request merged**